.eslintrc.js

```diff
@@ -6,16 +6,24 @@
 'use strict'
 
 module.exports = {
-  extends: '@newrelic',
+  extends: ['@newrelic', 'plugin:jsdoc/recommended'],
+  plugins: ['jsdoc'],
   rules: {
-    'consistent-return': 'off'
+    'consistent-return': 'off',
+    'jsdoc/require-jsdoc': 'off'
   },
   overrides: [
     {
       files: ['newrelic.js'],
       rules: {
         'header/header': ['off']
       }
+    },
```

Comment:
@garbados there was def some miscommunication happening via this PR comments.  I just went ahead and addressed what I was trying to explain.  Let's go over this when you are online today